NAMESPACE

```diff
@@ -51,6 +51,7 @@ export(tile1)
 export(translationalEff)
 export(txNames)
 export(txNamesWithLeaders)
+export(txSeqsFromFa)
```

Comment:
completely unnecessary export? there exist function readDNAStringSet

Reply:
Okey to keep this then ?